icons/loader-circle.json

```
@@ -0,0 +1,23 @@
+{
+  "$schema": "../icon.schema.json",
```

I think you need to add this if you move things but @karsa-mistmere probably knows more.
```suggestion
  "$schema": "../icon.schema.json",
  "aliases": [
    "loader-2"
  ],
```

---

> I think you need to add this if you move things but @karsa-mistmere probably knows more.

@jguddas I thought about that, but not sure in cases where the alias name is another existing icon…? @karsa-mistmere

---

@jguddas is right, we need aliases for backwards compatibility. And you've renamed `loader-2` to `loader-circle`, so `loader-2` used to be _this_ icon, not another one. :)